src/doc/gui_manual/Operators/OperatorTypes/PICS/LimitCycle_operator.rst

```diff
@@ -0,0 +1,196 @@
+.. _Limit Cycle operator:
+
+Limit Cycle operator
+~~~~~~~~~~~~~~~~~~~~
+
+The Limit Cycle Operator allows one to search for closed material lines (limit
+cycles) which are elliptic LCS's. To perform this search one defines a Poincaré
```

Comment:
As above, a lot of verbiage here and not sure if it really helps user with understanding of the controls/interface. And, if there is a nice on-line ref that elaborates, it would be useful to link to here.